SocketIOClientSwift/WebSocket.swift

```diff
@@ -378,21 +378,17 @@ public class WebSocket : NSObject, NSStreamDelegate {
         }
         if totalSize > 0 {
             if validateResponse(buffer, bufferLen: totalSize) {
-                dispatch_async(queue, { [weak self] in
```

Comment:
I believe these weak references are on purpose, otherwise there's a case where the WebSocket won't be released.